docs/api/public_php_api.md

```diff
@@ -104,84 +104,75 @@ $ cd /path/to/ezplatform
 
 Then use the app/console application with the `generate:bundle` command to start the bundle generation wizard.
 
-Let's follow the instructions provided by the wizard. Our objective is to create a bundle named `EzSystems/Bundles/CookBookBundle`, located in the `src` directory.
+Follow the instructions provided by the wizard. Your objective is to create a bundle named `EzSystems/Bundles/CookBookBundle`, located in the `src` directory.
 
 ``` bash
 $ php app/console generate:bundle
 ```
 
-The wizard will first ask about our bundle's namespace. Each bundle's namespace should feature a vendor name (in our own case: `EzSystems`), optionally followed by a sub-namespace (we could have chosen to use `Bundle`), and end with the actual bundle's name, suffixed with Bundle: `CookbookBundle`.
+The wizard will first ask about your bundle's namespace. Each bundle's namespace should feature a vendor name (for example: `EzSystems`), optionally followed by a sub-namespace (for example: `Bundle`), and end with the actual bundle's name, suffixed with bundle: `CookbookBundle`.
 
 **Bundle namespace**
 
-```
```

Comment:
This is the actual output when using the `generate:bundle` command, so it should either remain in code block/quote (and then not modified at all), or be rewritten to prose